ginicoeff.m

```diff
@@ -42,6 +42,7 @@
 
 % Author: Oleg Komarov (oleg.komarov@hotmail.it)
 % Tested on R14SP3 (7.1) and on R2009b
```

Comment:
If you have tested it on a later version of matlab and you are gonna amend the history, would be good if you added your version too.